subsys/bluetooth/common/Kconfig

```diff
@@ -77,10 +77,21 @@ config BT_BUF_ACL_RX_SIZE
 	  In a Controller only build this will determine the maximum ACL size
 	  that the Controller will send to the Host.
 
+config BT_BUF_ACL_RX_COUNT_EXTRA
+	int "Number of extra incoming ACL data buffers"
+	default 0
+	range 0 65535
+	help
+	  Number of incoming extra ACL data buffers sent from the Controller to
+	  the Host.
+
+	  By default, the number of incoming ACL data buffers is equal to
+	  CONFIG_BT_MAX_CONN + 1.
+
 config BT_BUF_ACL_RX_COUNT
-	int "Number of incoming ACL data buffers"
-	default 6
-	range 2 256
+	int "[DEPRECATED] Number of incoming ACL data buffers"
+	default 0
+	range 0 256
```

Comment:
```suggestion
	int "[DEPRECATED] Number of incoming ACL data buffers"
	select DEPRECATED
	default 0
	range 0 256
```

Reply:
This was discussed earlier. It would select DEPRECATED even in the default case where this wasn't explicitly set anywhere, which is not what we want. That's why there's instead a `#warning` in the code when it's not 0.

Reply:
Yes and more to that, it's not possible to select it because it's not a `bool`. I am not familiar enough with Kconfig to explain why.

Reply:
Yeah, I don't think Kconfig supports `depends on X if Y` or `select X if Y`. 

It is possible with a proxy bool I think, so
```
config FOO_BOOL
    def_bool FOO_VAL > 0
    select DEPRECATED

config FOO_VAL
    int "Foo"
```

may work